### ISSUE TITLE
Improve Bootstrap theme color contrast

### ISSUE THREAD

**Issue by jonearley** [projecthydra/sufia#2091]
### Descriptive summary

Many of the Bootstrap theme colors we use fail to meet WCAG AA color contrast standards. 

### Solution

Let's improve Bootstrap theme color palette contrast ratios. The Bootstrap community has proposed a few options already that we could adopt. Implementation should be as easy as updating a few variables.

### Related Work

This issue was a result of discussions here: https://github.com/projecthydra/sufia/issues/2076.

Related Bootstrap issues/pulls:

- https://github.com/twbs/bootstrap/pull/19693
- https://github.com/twbs/bootstrap/issues/3572

**Comment by cbeer** [comment]
Why does this need to go into Sufia (instead of waiting for upstream changes, or deferring to the local applications)?

**Comment by jonearley** [comment]
@cbeer Good point. Would moving this issue to curation concerns make more sense?
